### DISCUSSION TITLE
Squirrel chrage

### DISCUSSION THREAD

**Discussion by Azgaar** [Azgaar/Armoria#110] [Ideas]
@Blipz, what about replacing the squirrel with this one? - https://upload.wikimedia.org/wikipedia/commons/5/5c/Coa_England_Family_Rosalinda_Lucinda_Mandragon_de_la_Vega.svg

The one from WappenWiki looks super ugly. It's like a Chernobyl squirrel.

Also did you see the list of suggested charges? https://docs.google.com/spreadsheets/d/1jhwhRnut3ubPQ9cO3ph1wDwLPvWC607U_LjHA8_-YMg/edit#gid=0

**Comment by Blipz** [comment]
Yes, I saw the list, lot of work to be done!

I can definitely try the other squirrel image.

**Comment by Blipz** [comment]
Done [here](https://github.com/Azgaar/Armoria/commit/13f7969054f3135bf72b8fe0cd83c1e2192f6026), with some other charges.

**Reply by Azgaar** [reply]
That's is cool

**Comment by Blipz** [comment]
Merged in 7007500ba8bd6e82ae6145a41e9a13510d9ee679.
